modules/libcom/src/as/asLibRoutines.c

```diff
@@ -29,6 +29,8 @@
 #include "postfix.h"
 #include "asLib.h"
 
+#undef ECHO /* from termios.h */
```

Comment:
What changes here? I don't see this macro used anywhere.

Reply:
Ok, but why is it a problem here? I don't see it used in this file. Unless emacs end grep are making fun of me.

Reply:
I would say flex is making fun of you. The warning I get is:
```
In file included from ../as/asLib.y:205:0:
asLib_lex.c:26:0: warning: "ECHO" redefined [enabled by default]
/opt/eldk-5.1/powerpc-4xx-softfloat/sysroots/ppc405-linux/usr/include/bits/termios.h:192:0: note: this is the location of the previous definition
```
asLib_lex.c  is a generated file made from modules/libcom/src/flex/flex.skel.static.